docs/conf.py

```diff
@@ -213,6 +215,16 @@ def rstjinja(app, docname, source):
 napoleon_use_rtype = False
 # Disable google style docstrings
 napoleon_google_docstring = False
+nitpick_ignore = [('py:class', 'optional'),
+                  # Prevents shpinx nitpicky mode picking up on optional
+                  # (see https://github.com/sphinx-doc/sphinx/issues/6861)
+                  ('py:obj', 'numpy.datetime64'),
+                  # See https://github.com/numpy/numpy/issues/10039
+                  ('py:class', 'sunpy.coordinates.frames.SunPyBaseCoordinateFrame'),
+                  ('py:class', 'sunpy.coordinates.frames.BaseHeliographic'),
+                  # Class inheritance will try to link to these, but we don't want
+                  # base coordinate frames in the docs, so ignore
```

Comment:
They probably *should* show up somewhere in our docs...  However, that can be dealt with separately.

Reply:
👍 was going to check with you at some point - presumably if they should be in the docs they just need adding to the appropriate `__all__`.